Add limitation 'Event Type is Page' (#5503)
segmentio/segment-docs: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md

### Data is not sent to Google

In order for data to be sent downstream to Google Analytics, check your mappings to ensure that:
1. The **setConfigurationFields** mapping is enabled in your mappings.
2. You've added at least one other event mapping for an event you want to send to Google Analytics.

> 3. Your event mapping doesn't use the condition 'Event Type is Page'. The GA config command uses a Page event when your page loads, so it can't be used for triggering additional events in GA4. Segment recommends using a [Track call](/docs/connections/spec/track/) for any additional events you'd like to appear in GA4. This ensures your events go through after the initial 'config' loads.

Reviewer: ```suggestion
3. Your event mapping doesn't use the condition 'Event Type is Page'. The GA config command uses a Page event when your page loads, so Page calls can't be used for triggering additional events in GA4. Segment recommends using a [Track call](/docs/connections/spec/track/) for any additional events you'd like to appear in GA4 so your events can go through after the initial 'config' loads.
```

> 3. Your event mapping doesn't use the condition 'Event Type is Page,' which depends on the page event that's triggered when your site loads. The initial 'Page' event is claimed by the GA config command, so it can't be used for triggering additional events in GA4. We recommend using [analytics.track()](https://segment.com/docs/connections/spec/track/) for any extra events you'd like to appear in GA4. This ensures your events go through after the initial 'config' has loaded.

Reviewer: ```suggestion
3. Your event mapping doesn't use the condition 'Event Type is Page'. The GA config command uses a Page event when your page loads, so it can't be used for triggering additional events in GA4. Segment recommends using a [Track call](/docs/connections/spec/track/) for any extra events you'd like to appear in GA4. This ensures your events go through after the initial 'config' has loaded.
```
